docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -420,6 +423,26 @@ For example, the following command runs fixtures whose names match `Page.*`. The
 testcafe ie my-tests -F "Page.*"
 ```
 
+### --test-meta \<key=value\[,key2=value2,...\]\>
+
+TestCafe runs tests whose [metadata](../test-api/test-code-structure.md#specifying-testing-metadata) [matches](https://lodash.com/docs/#isMatch) the specified key-value pair.
+
+For example, the following command runs tests whose metadata has the `device` property set to the `mobile` value and the `env` property set to the `production` value.
```

Comment:
For example, the following command runs tests whose metadata has the `device` property set to the `mobile` value and the `env` property set to the `production` value -> For example, the following command runs tests whose metadata's `device` property is set to `mobile`, and `env` property is set to `production` .